doc/api_review/core/kdbClose.md

```diff
@@ -0,0 +1,79 @@
+- start = 2021-01-23 18:10
+- end = 2021-01-23 18:10
+- reviewer = Stefan Hanreich <stefanhani@gmail.com>
+
```

Comment:
synopsis and/or version will be needed as the same function name can occur several times due to symbol versioning